src/user/types.ts

```diff
@@ -8,6 +8,7 @@ export type NewUser = {
   nickname: string,
   created: Date,
   type: UserType,
+  active: boolean
 };
```

Comment:
Also a nice way to declare the NewUser type is something like this:

`export type NewUser = Omit<User, 'id'>;`

It's the same type as the user, except you don't have an id yet. The id can be appended to the record once the record is inserted into the db.